tox.ini

```diff
@@ -25,16 +25,18 @@ commands = python-codacy-coverage -r coverage.xml
 [testenv:publish]
 changedir =
 passenv =
-  TWINE_*
+    TWINE_*
+    CI_COMMIT_*
 skip_install=true
 deps =
-    setuptools_scm
+    setuptools_scm==8.0.4
+    setuptools_git_versioning<2
```

Comment:
maybe we should pin it in pyproject.toml instead?

Reply:
already in there, second line.